README.md

```diff
@@ -1,7 +1,7 @@
 # bpmodels
-
-[![Travis-CI Build Status](https://travis-ci.org/sbfnk/bpmodels.svg?branch=master)](https://travis-ci.org/sbfnk/bpmodels)
-[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/y37i8x0wo9o8s2wf?svg=true)](https://ci.appveyor.com/project/sbfnk/bpmodels)
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/sbfnk/bpmodels/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/sbfnk/bpmodels/actions/workflows/R-CMD-check.yaml)
+<!-- badges: end -->
```

Comment:
Could you:
1. Edit the URL to reflect that `{bpmodels}` is now under the Epiverse TRACE org? The repo does redirect correctly, but just in case.
2. Move the Codecov badge inside the badges section? Very minor issue.

Reply:
Done!